### ISSUE TITLE
Create Script that copies all content from xiaofang internal drive to SD card

### ISSUE THREAD

**Issue by qualytest** [samtap/fang-hacks#247]
Hi Guys,

Is that possible using the snx_autorun.sh file?
This will allow to back up the camera drive before attempting any modifications.

Thanks

**Comment by samtap** [comment]
Yes it's possible but creating a flashable fw image is more involved, not my expertise. You should use dd to create backups of /dev/mtdblock0-5 and combine them into a single firmware file, probably adding a header and some padding here and there?
